pkg/storage/snapshot/restore.go

```diff
@@ -363,6 +401,8 @@ func (t *vmRestoreTarget) UpdateRestoreInProgress() error {
 
 		// unfortunately, status Updater does not return the updated resource
 		// but the controller is watching VMs so will get notified
+		// TODO: if this is true then don't we need to exist the reconcile loop
+		// to make sure we are working on an updated vm?
```

Comment:
I think it only matters if the VM will be updated again in the same reconcile loop, that should not be the case

Reply:
actually now that no longer using the "status updater" abstraction can get rid of comment and get updated vm returned